extension/src/popup/constants/metricsNames.ts

```diff
@@ -1,4 +1,5 @@
 export const METRIC_NAMES = {
+  viewDebug: "loaded screen: debug",
```

Comment:
This should never actually make it into amplitude, but special casing it in the route metrics logic seemed bad